src/App.js

```diff
@@ -213,14 +208,14 @@ class App {
 						Object.assign( tokenInfo, successData );
 
 						setTimeout( () => {
-							this.revisionPopup.show( tokenInfo, $( e.target ) );
+							this.revisionPopup.show( tokenInfo, $( e.target ), isCached );
```

Comment:
(Note a blocker for this, just more a comment in general)
This is one of the reasons I think the wwtController should do this internally; sending over the internal "isCached" state seems iffy to me.

This patch isn't the first thing that does it, but when we do refactor, we should take another look at how to do it without sending this internal state everywhere.